src/test/java/com/google/api/codegen/testdata/library.proto

```diff
@@ -109,7 +109,12 @@ service LibraryService {
   }
 
   // Gets a book from an archive.
-  rpc GetBookFromArchive(GetBookFromArchiveRequest) returns (Book) {
+  rpc GetBookFromArchive(GetBookFromArchiveRequest) returns (BookFromArchive) {
+    option (google.api.http) = { get: "/v1/{name=archives/**/books/*}"};
```

Comment:
Sot turns out that ** is only supposed to be valid at the end of a resource name pattern.